wrappers/services.go

```diff
@@ -224,6 +224,34 @@ func StopServices(apps []*snap.AppInfo, inter interacter) error {
 		if !app.IsService() || !osutil.FileExists(app.ServiceFile()) {
 			continue
 		}
+		// Skip stop on refresh when refresh mode is "keep"
+		if reason == snap.StopReasonRefresh {
+			switch app.RefreshMode {
+			case "keep":
```

Comment:
I'm sorry for the back and forth on this term, Michael. I'm still concerned about the current one.. the problem with both survive and keep is that it feels to innocent. I'd like to have a term that gives people a strange feeling about using it, and make them look for details on why that is so. As hopefully my last suggestion here, how about _"endure"_. This comes close to _"survive",_ but it has that feeling that it might not be that easy to keep it up, which is realistic given the constraints we've discussed for this (things changing behind the process, write access being lost, etc).

Reply:
No worries, I'm happy about the attention to this. I changed this now in my PR to `endure`.